src/content/docs/en/reference/content-loader-reference.mdx

```diff
@@ -99,7 +99,7 @@ By default it uses [`github-slugger`](https://github.com/Flet/github-slugger) to
 <Since v="5.0.0" />
 </p>
 
-The `file()` loader creates entries from a single file that contains an array of objects with a unique `id` field, or an object with string keys. It supports JSON or YAML, and you can provide a custom `parser` for data files it cannot parse by default.
+The `file()` loader creates entries from a single file that contains an array of objects with a unique id field, or an object with `id`'s as keys and `entries` as values. It supports JSON or YAML, and you can provide a custom `parser` for data files it cannot parse by default.
```

Comment:
```suggestion
The `file()` loader creates entries from a single file that contains an array of objects with a unique `id` field, or an object with IDs as keys and entries as values. It supports JSON or YAML, and you can provide a custom `parser` for data files it cannot parse by default.
```
I think the unique `id` field should be written as code, but I don't think `entries` is a special property needed by the object, so I think the rest is just in plain language.